expected/init_test_healpix.out.in

```diff
@@ -1,2 +1,2 @@
-psql:pg_sphere.test.sql:9181: NOTICE:  return type smoc is only a shell
-psql:pg_sphere.test.sql:9187: NOTICE:  argument type smoc is only a shell
+psql:pg_sphere.test.sql:9167: NOTICE:  return type smoc is only a shell
+psql:pg_sphere.test.sql:9173: NOTICE:  argument type smoc is only a shell
```

Comment:
You are almost there! Change "9167" to "9195" on line 1. Change "9173" to "9201" on line 2.